docs/examples/operator_spaceship__const_reference.cpp

```diff
@@ -24,8 +34,8 @@ int main()
 
 
     // output values and comparisons
-    std::cout << array_1 << " <=> " << array_2 << " := " << to_string(array_1 <=> array_2) << '\n';
-    std::cout << object_1 << " <=> " << object_2 << " := " << to_string(object_1 <=> object_2) << '\n';
-    std::cout << string << " <=> " << number << " := " << to_string(string <=> number) << '\n';
-    std::cout << string << " <=> " << discarded << " := " << to_string(string <=> discarded) << '\n';
+    std::cout << array_1 << " <=> " << array_2 << " := " << to_string(array_1 <= > array_2) << '\n';
+    std::cout << object_1 << " <=> " << object_2 << " := " << to_string(object_1 <= > object_2) << '\n';
+    std::cout << string << " <=> " << number << " := " << to_string(string <= > number) << '\n';
+    std::cout << string << " <=> " << discarded << " := " << to_string(string <= > discarded) << '\n';
```

Comment:
These spaces look unintended, also in the other file.

Reply:
Right, they need `// *NOPAD*` or something. I ran `make pretty` after CI failed because I forgot …